homeassistant/components/bluesound/media_player.py

```diff
@@ -917,7 +921,15 @@ async def async_set_shuffle(self, shuffle):
         """Enable or disable shuffle mode."""
         value = "1" if shuffle else "0"
         return await self.send_bluesound_command(f"/Shuffle?state={value}")
-
+    
+    async def async_next_input(self):
+        """Switch to next available input."""
+        return await self.send_bluesound_command("/ExternalSource?id=%2B&schemaVersion=26")
```

Comment:
We shouldn't extend this integration until the device specific interface code and data parsing has been extracted to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices